### PR TITLE
fix(types): adding correct redirect-home props to fix some ts-ignores

### PR DESCRIPTION
**Pull request by mateoguzmana**
Checklist:
- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Removing `RedirectHome` component passed props to fix some ts-ignores

### CODE REVIEW

File: client/src/pages/certification.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ function Certification(): JSX.Element {
         path={withPrefix('/certification/:username/:certSlug')}
       />
 
-      <RedirectHome default={true} />
+      <RedirectHome />
```
Comment (ojeytonwilliams) [inline client/src/pages/certification.tsx:20]:
```suggestion
      <RedirectHome default />
```
We have to pass this prop for the component to be rendered. But `default={true}` is a little ugly and this seems nicer :shrug:

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Hey @mateoguzmana thanks for putting the time in to tidy up the code base. Unfortunately, we do need these props.

I think the following changes should make the code a little less confusing:

**Comment by mateoguzmana** [comment]
> Hey @mateoguzmana thanks for putting the time in to tidy up the code base. Unfortunately, we do need these props.
> 
> I think the following changes should make the code a little less confusing:

I had the feeling this was somehow used :D, thanks a lot for clarifying @ojeytonwilliams! Fixed

**Review by ojeytonwilliams — APPROVED** [review]
Thanks, @mateoguzmana! LGTM 👍

**Review by Sboonny — APPROVED** [review]
LGTM

**Pull request merged**